packages/web/components/bridge/immersive/use-bridge-quotes.ts

```diff
@@ -377,28 +385,22 @@ export const useBridgeQuotes = ({
           `${providerId}${JSON.stringify(params)}`,
           inputCoin.trim(true).toString(),
           isWithdraw,
-          toAddress ?? "" // use osmosis account (destinationAddress) for account keys (vs any EVM account)
+          (isWithdraw ? fromAddress : toAddress) ?? "" // use osmosis account (destinationAddress) for account keys (vs any EVM account)
```

Comment:
Transfer history is keyed and filtered by Osmosis account that is currently connected